tests/puppeteer/README.md

```diff
@@ -3,49 +3,77 @@
 ## How to install your environment
 
 ```bash
+# Clone Prestashop
 git clone https://github.com/PrestaShop/PrestaShop/
+# Install dependencies in puppeteer folder
 cd tests/puppeteer/
-npm i
+npm install
 ```
 
-## LinkChecker
-This script will detect not found and erroneous pages, by crawling your back office and front office
-
-### Available command line parameters
-
+## Available command line parameters
 | Parameter           | Description      |
 |---------------------|----------------- |
-| URL_BO              | URL of your PrestaShop website Back Office (default to **http://localhost:8080/admin-dev/**) |
-| URL_FO              | URL of your PrestaShop website Front Office (default to **http://localhost:8080/**) |
-| LOGIN               | LOGIN of your PrestaShop website (default to **demo@prestashop.com**) |
-| PASSWD              | PASSWD of your PrestaShop website (default to **prestashop_demo**) |
+| URL_FO              | URL of your PrestaShop website Front Office (default to **`http://localhost:8080/`**) |
+| URL_BO              | URL of your PrestaShop website Back Office (default to **`URL_FO + admin-dev/`**) |
+| URL_INSTALL         | URL of your PrestaShop website Front Office (default to **`URL_FO + install-dev/`**) |
+| LOGIN               | LOGIN of your PrestaShop website (default to **`demo@prestashop.com`**) |
+| PASSWD              | PASSWD of your PrestaShop website (default to **`prestashop_demo`**) |
+| SHOPNAME            | Shop Name of tour PrestaShop (default to **`Prestashop`**) |
+| DB_USER             | Login user of your MySql (default to **`root`**) |
+| DB_PASSWD           | Password for your MySql (default to **`empty`**) |
+| HEADLESS            | Boolean to run tests in headless or not (default to **`true`**) |
+
+Before Running tests, you should install your shop manually or run the install script **`campaigns/sanity/01_installShop/*`** with [specific test command](README.md#specific-test).
 
-### Launch script
-If you want to run the links checker test you can run the script **test/linkchecker.js**
+## Sanity tests 
+This campaign includes a non-exhaustive set of tests and it will ensure that the most important functions work.
+
+### Launch all scripts
+If you want to run all sanity tests, you can run scripts in **`campaigns/sanity/*`**
 
 #### With default values
 
 ```bash
-npm run linkchecker
+npm run sanity-tests
 ```
 
 #### With custom values
+You can add parameters that you need in the beginning of your command 
+```bash
+HEADLESS=false URL_BO="Your_Shop_URL_BO" URL_FO="Your_Shop_URL_FO" npm run sanity-tests
+```
+
+### Stop tests when first step in failed
+If you want to run all sanity tests, and stop tests when first step is failed, you can use the travis command for sanity tests
```

Comment:
Maybe explain what it changes ? Like

This command add the Mocha `--bail` parameter which stops the campaign when the first test fails.